src/ray/raylet/client_connection_test.cc

```diff
@@ -35,18 +35,21 @@ class ClientConnectionTest : public ::testing::Test {
   ClientConnectionTest()
       : io_service_(), in_(io_service_), out_(io_service_), error_message_type_(1) {
 #if defined(BOOST_ASIO_HAS_LOCAL_SOCKETS)
-    boost::asio::local::connect_pair(in_, out_);
+    boost::asio::local::stream_protocol::socket input(io_service_), output(io_service_);
```

Comment:
Hmm, why is this change needed

Reply:
You can't `connect_pair` generic sockets, it's a `boost::asio::local` thing. (It doesn't compile otherwise.)